extensions/wikia/ContributionPrototype/scripts/LoadEditor.js

```diff
@@ -0,0 +1,71 @@
+require(['jquery', 'wikia.window'],
+	function ($, window) {
+		'use strict';
+
+		$(function() {
+			console.log("CP load editor");
+
+			var plugins = [];
+			var config = {
+				body: $('<div/>'),
+				element: $('<div/>'),
+				mode: 'source'
+			}
+
+			window.WikiaEditor.create(plugins,config);
+
+		});
+
+		function init(window) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "window". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)